src/modules/spots/useSpots.js

```diff
@@ -269,6 +283,7 @@ const useSpots = (props) => {
     getSpotsSortedReverseChronologically: getSpotsSortedReverseChronologically,
     getAllImageBaseMaps: getAllImageBaseMaps,
     findRootSpot: findRootSpot,
+    getSpotByImageId: getSpotByImageId,
```

Comment:
Make sure this list stays alphabetized. And the order of the functions themselves above should be alphabetized as well. We've all gotten away from doing this sometimes but trying to be better.